### DISCUSSION TITLE
Best module for dynamic fracture nucleation and propagation without a notch in the geometry

### DISCUSSION THREAD

**Discussion by miraboreasumr** [idaholab/moose#27654] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [x] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

XFEM, Peridynamics or Phase Field, consider coupling with flow
Edited: able to output the porosity, permeability change after the fracturing

**Comment by GiudGiud** [comment]
@bwspenc or @laagesen 

I would guess peridynamics

**Reply by miraboreasumr** [reply]
more suggestions?

**Reply by GiudGiud** [reply]
@gambka

**Reply by gambka** [reply]
Is this a 3D or 2D geometry? I think any approaches would work but Peridynamics may have some difficulties coupling to flow given that it is a bond based formulation and not a finite element formulation like XFEM and phase-field. That being said I am not aware of any coupling of these methods to flow in MOOSE. In terms of the quantities of interest those would not be computed directly and you would need to add postprocessors to extract that information.

**Reply by jiangwen84** [reply]
If you are asking which approach is more appropriate for your problem,  you should read papers to understand each method's pros and cons.  If you would like to know the muturity of each method implementation in moose to handle dynamic fracture,  pf>pd>xfem. 

Even without notch,  you need to inreduce initial flaw,  e.g purturb property or have gemomtry defect

**Reply by miraboreasumr** [reply]
> That being said I am not aware of any coupling of these methods to flow in MOOSE.

Thanks, sorry I didn't get this "That being said I am not aware of any coupling of these methods to flow in MOOSE." They are not coupled?

**Reply by GiudGiud** [reply]
If you define multiple equations in the same input file. they will naturally be fully coupled.
We are not aware of any work doing this before

I dont expect coupling to flow simulations through porosity to be challenging numerically? Maybe it will be right at the formation of cracks. 

Edit: there has been a lot of work in moose in having equations being defined on a domain that grows or shrinks. The flow equations could be active only on part of the domain, where the porosity is such that flow is possible

**Reply by miraboreasumr** [reply]
> If you are asking which approach is more appropriate for your problem, you should read papers to understand each method's pros and cons. If you would like to know the muturity of each method implementation in moose to handle dynamic fracture, pf>pd>xfem.
> 
> Even without notch, you need to inreduce initial flaw, e.g purturb property or have gemomtry defect

Thanks, just want to clear, when you mentioned geometry defect, does Bazilian test belong to this?
